releasenotes/notes/0.17/bool_expression_phaseoracle-1802be3016c83fa8.yaml

```diff
@@ -9,7 +9,7 @@ features:
     .. jupyter-execute::
```

Comment:
I don't know if releasenotes from past releases will be regenerated, @mtreinish or @jakelishman will certainly know! I would think they are not, in which case it would probably be better to not change this?

Reply:
They do get regenerated as part of the build, always. When old notes start causing trouble (because of deprecations, etc), we change them from `.. jupyter-execute::` blocks to `.. code-block:: python` blocks, so the code as it was originally written is still there, but doesn't kill the build.

Reply:
Ya, the code in the old release note was making an error in the CI, so I changed it.